src/frontends/sql/BUILD

```diff
@@ -71,6 +72,7 @@ cc_library(
 cc_test(
     name = "decoder_context_test",
     srcs = ["decoder_context_test.cc"],
+    copts = ["-fexceptions"],
```

Comment:
It is because `std::stoul` throws an exception when it fails to parse a number from a string. I use `std::stoul` in this test and want to make sure it does not fail to parse a string as a number, and so I use `EXPECT_NO_THROW`, which requires exceptions to be enable.

Reply:
It looks like absl has a non-exception-throwing equivalent I can use, will refactor to eliminate `-fexceptions`.

Reply:
Done.